nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -31,12 +32,20 @@ in
     services.slurm = {
 
       server = {
-        enable = mkEnableOption "slurm control daemon";
-
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Wether to enable the slurm control daemon.
+            Note that the standard authentication method is "munge".
+            The "munge" service needs to be setup up in order for
+            slurm to work properly (see <literal>services.munge<literal>).
```

Comment:
Would it be a bad idea to enable `munge` in turn?

Reply:
That would be a good idea. However, you need to provide a key file to munge module to make work.